OMCompiler/Compiler/NBackEnd/Modules/2_Pre/NBInline.mo

```diff
@@ -210,6 +210,7 @@ public
     else
       if Flags.isSet(Flags.FAILTRACE) then
         Error.addCompilerWarning("Failed to inline following equation:\n" + Equation.toString(eqn));
+        changed := false;
```

Comment:
I think you need to put this outside the `if`.

Reply:
true!